django_x509/admin.py

```diff
@@ -1,7 +1,18 @@
 from django.contrib import admin
 
-from .base.admin import CaAdmin, CertAdmin
+from .base.admin import BaseX509Form, CaAdmin, CertAdmin
+from .base.models import BaseX509
 from .models import Ca, Cert
 
+
+class Form(BaseX509Form):
+    class Meta(BaseX509Form.Meta):
+        model = BaseX509
+
+
+class CertAdminForm(CertAdmin):
```

Comment:
Change `CertAdminForm` to `CertAdmin` and probably we'll have to change the import name to `AbstractCertAdmin`